resources/views/layouts/form.blade.php

```diff
@@ -45,7 +45,7 @@
         }
     </style>
     <script type="text/javascript" src="{{ asset('vendor/jsvalidation/js/jsvalidation.js')}}"></script>
-    {!! $validator !!}
+    {{ $validator }}
```

Comment:
This probably needs to be `{!! $validator !!}` as it is generating Javascript, I think. The [documentation](https://github.com/proengsoft/laravel-jsvalidation/wiki/Basic-Usage) also shows it as using `{!! $validator !!}`. Thanks!